contrib/kind-common

```diff
@@ -649,3 +649,61 @@ get_kubevirt_release_url() {
 
     echo "$kubevirt_release_url"
 }
+
+readonly FRR_K8S_VERSION=v0.0.14
+readonly FRR_TMP_DIR=$(mktemp -d -u)
+
+clone_frr() {
+  [ -d "$FRR_TMP_DIR" ] || {
+    mkdir -p "$FRR_TMP_DIR" && trap 'rm -rf $FRR_TMP_DIR' EXIT
+    pushd "$FRR_TMP_DIR" || exit 1
+    git clone --depth 1 --branch $FRR_K8S_VERSION https://github.com/metallb/frr-k8s
+    popd || exit 1
+  }
+}
+
```

Comment:
I did not. I guess it could potentially be used but that's geared towards metallb tests and does a lot more than what I need, and on the other side I am setting up FRR in a different way than they do. Can we defer this until I see to e2e testing integrated with metallb?